cmd/bootstrap/cmd/machine_account_test.go

```diff
@@ -0,0 +1,120 @@
+package cmd
+
+import (
+	"fmt"
+
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/model/bootstrap"
+	model "github.com/onflow/flow-go/model/bootstrap"
+	ioutils "github.com/onflow/flow-go/utils/io"
+)
+
+func TestMachineAccountHappyPath(t *testing.T) {
+	var machineAccountHappyPath = `^read machine account private key json` +
+		`encoded public machine account key` +
+		`wrote file /tmp/%s/private-root-information/private-node-info_\S+/node-machine-account-info.priv.json`
+
+	dirName := strconv.FormatInt(time.Now().UnixNano(), 10)
+	regex := regexp.MustCompile(fmt.Sprintf(machineAccountHappyPath, dirName))
+
+	// command flags
+	flagOutdir = fmt.Sprintf("/tmp/%s", dirName)
```

Comment:
We can use the test utility here
https://github.com/onflow/flow-go/blob/ff92e3b5a8edfb80eb797e8896c8c2a0bdea8fcd/utils/unittest/unittest.go#L172